### PR TITLE
scraper code and results added

### PR DESCRIPTION
**Pull request by MuskanSinghal**
# Sweet! You've got a Pull Request (PR)! 

# What does this PR do?

### CODE REVIEW

File: myfile.csv [hunk 1]
```diff
@@ -1 +1 @@
...
+"1","2","3"
```
Comment (claytonmgravatt) [inline myfile.csv:1]: This file is causing a merge conflict and may need to be removed from the PR.

File: maindata.py [hunk 2]
```diff
@@ -4,6 +4,8 @@
 from csvoutput import CsvOutput
 from wikipediafunction import WikipediaFunction
 from youtubefunction import YoutubeFunction
+import web_scraper as webScraper
```
Comment (claytonmgravatt) [inline maindata.py:7]: I believe this maindata.py is an older version than what's currently in the main repo and therefore is causing a merge conflict. You might want to sync up with the most recent files in the main branch and resubmit your maindata.py

File: web_scraper.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    chrome_options = Options()
+    chrome_options.add_argument("--headless")
+    chrome_options.add_argument("--window-size=1920,1080")
+    chrome_options.add_argument("--disable-gpu")
+    chrome_options.add_argument("--disable-extensions")
+    chrome_options.add_experimental_option("useAutomationExtension", False)
+    chrome_options.add_argument("--proxy-server='direct://'")
+    chrome_options.add_argument("--proxy-bypass-list=*")
+    chrome_options.add_argument("--start-maximized")
+    chrome_options.add_argument("--headless")
```
Comment (claytonmgravatt) [inline web_scraper.py:21]: Great! I like that you're making it headless! Did this argument need to be added twice?

**Review by claytonmgravatt — CHANGES_REQUESTED** [review]
Looks fantastic, this is all very well done. Only changes requested are the merge conflicts.

**Comment by gregv** [comment]
@MuskanSinghal its good practice to close pull requests (by merging code) within a week - other people won’t see changes in pull requests so the faster they are merged, the better

**Pull request closed without merging**